horovod/common/parameter_manager.cc

```diff
@@ -388,11 +398,17 @@ ParameterManager::BayesianParameter::BayesianParameter(
     test_points_(test_points),
     iteration_(0) {
   ResetBayes();
+  Reinitialize(FilterTestPoint(0));
   ResetState();
 }
 
 void ParameterManager::BayesianParameter::SetValue(BayesianVariable variable, double value, bool fixed) {
   if (fixed) {
+    int32_t index = index_[variable];
```

Comment:
Scratching my head about what does this do - could you add comments?

Reply:
Added comments, let me know if that helps clear it up.